data/icon-overlay.js

```diff
@@ -0,0 +1,90 @@
+const host = window.location.host;
+
+// YOUTUBE HOME PAGE
+// TODO: listen for infinite scrollesque updates and apply this again
+// if necessary
+const ytHomeContainers = Array.from(document.querySelectorAll('#feed .yt-lockup-thumbnail'));
+if ((host.indexOf('youtube.com') > -1) && ytHomeContainers.length) {
+  ytHomeContainers.forEach(el => {
+    el.classList.add('minvid__overlay__wrapper');
+    const tmp = getTemplate();
+    tmp.addEventListener('click', function(ev) {
+      evNoop(ev);
+      const partialUrl = el.querySelector('.yt-uix-sessionlink').getAttribute('href');
+      if (partialUrl.length) {
```

Comment:
Seems weird to check the length of a string. Maybe just `if (partialUrl)`, since it does the same thing?